lib/rdf2marc/rdf2model/mappers/leader.rb

```diff
@@ -7,7 +7,7 @@ module Mappers
       class Leader < BaseMapper
         def generate
           {
-            record_status: item.admin_metadata.query.path_first_literal([[BF.status, BF.Status], BF.code]),
+            record_status: record_status,
```

Comment:
My suggestion is to the rid of L10 and L20-31 so that no mapping is attempted for record status.

Reply:
(And sorry I have been repeatedly unclear.)

Reply:
@justinlittman OK, works for me.